R/plotAbundanceDensity.R

```diff
@@ -20,26 +20,34 @@
 #'   will be disabled.)
 #'   
 #' @param n a positive integer specifying the number of the most abundant taxa
-#'   to show. (default: \code{n = min(nrow(object), 25L)})
+#'   to show. (default: \code{n = min(nrow(x), 25L)})
 #'  
-#' @param colour_by a single character value defining a column from
+#' @param colour.by a single character value defining a column from
 #'   \code{colData}, that is used to color plot. Must be a value of
-#'   \code{colData()} function. (default: \code{colour_by = NULL})
+#'   \code{colData()} function. (default: \code{colour.by = NULL})
+#'   
+#' @param colour_by Deprecated. Use \code{colour.by} instead.
 #' 
-#' @param shape_by a single character value defining a column from
+#' @param shape.by a single character value defining a column from
 #'   \code{colData}, that is used to group observations to different point shape
-#'   groups. Must be a value of \code{colData()} function. \code{shape_by} is
-#'   disabled when \code{layout = "density"}. (default: \code{shape_by = NULL})
+#'   groups. Must be a value of \code{colData()} function. \code{shape.by} is
+#'   disabled when \code{layout = "density"}. (default: \code{shape.by = NULL})
+#'   
+#' @param shape_by Deprecated. Use \code{shape.by} instead.
 #' 
-#' @param size_by a single character value defining a column from
+#' @param size.by a single character value defining a column from
 #'   \code{colData}, that is used to group observations to different point size
-#'   groups. Must be a value of \code{colData()} function. \code{size_by} is
-#'   disabled when \code{layout = "density"}. (default: \code{size_by = NULL})
+#'   groups. Must be a value of \code{colData()} function. \code{size.by} is
+#'   disabled when \code{layout = "density"}. (default: \code{size.by = NULL})
+#'   
+#' @param size_by Deprecated. Use \code{size.by} instead.
 #' 
-#' @param order_descending \code{TRUE}, \code{FALSE} or\code{NA}: Should the
+#' @param order.descending \code{TRUE}, \code{FALSE} or\code{NA}: Should the
 #'   results be ordered in a descending order? If \code{NA} is given the order
-#'   as found in \code{object} for the \code{n} most abundant taxa is used.
-#'   (default: \code{order_descending = TRUE})
+#'   as found in \code{x} for the \code{n} most abundant taxa is used.
+#'   (default: \code{order.descending = TRUE})
+#'   
+#' @param order_descending Deprecated. Use \code{order.descending} instead.
 #'   
 #' @param ... additional parameters for plotting. 
 #' \itemize{
```

Comment:
You should be careful with additional parameters. In some functions, we use pass them to scater package. Should be checked that all these modifying parameters are in miaViz, and these modifications works (the unit tests should check that though),  Just wanted to comment so that you are aware of that